.github/renovate.json

```diff
@@ -6,6 +6,26 @@
   "ignorePaths": [
     "Src/Generated/**"
   ],
+  "packageRules": [
+    {
+      "matchPaths": [
```

Comment:
Any reason for not using multiple paths here, given that the rest of the rule is the same?

Reply:
Nope, just that I did it one by one and didn't thinkg to merge them at the end.
Changing now...

Reply:
Done. PTAL.